cirq/sim/density_matrix_simulator.py

```diff
@@ -120,14 +119,36 @@ def __init__(self,
                  *,
                  dtype: Type[np.number] = np.complex64,
                  noise: 'cirq.NOISE_MODEL_LIKE' = None,
-                 seed: value.RANDOM_STATE_LIKE = None):
+                 seed: value.RANDOM_STATE_LIKE = None,
+                 replace_measurement_with_dephasing: bool = False):
         """Density matrix simulator.
 
          Args:
             dtype: The `numpy.dtype` used by the simulation. One of
                 `numpy.complex64` or `numpy.complex128`
             noise: A noise model to apply while simulating.
             seed: The random seed to use for this simulator.
+            replace_measurement_with_dephasing: if True, then the simulation
+                will treat measurement as dephasing instead of collapsing
+                process.
+
+                Example:
+                q0 = cirq.LineQubit.range(1)
```

Comment:
Examples should be written in doctest format, so that they are tested.

```
Examples:
    >>> circuit = ...
    >>> print(...)
    expected output

    >>> more stuff
```

Reply:
Done